pkg/ddl/create_table.go

```diff
@@ -808,8 +814,8 @@ func handleTableOptions(options []*ast.TableOption, tbInfo *model.TableInfo) err
 				return dbterror.ErrUnsupportedShardRowIDBits
 			}
 			tbInfo.ShardRowIDBits = op.UintValue
-			if tbInfo.ShardRowIDBits > shardRowIDBitsMax {
-				tbInfo.ShardRowIDBits = shardRowIDBitsMax
+			if tbInfo.ShardRowIDBits > variable.MaxShardRowIDBits {
```

Comment:
So here the system variable has higher priority than user manually specific?

Reply:
No, manually setup has a higher priority.

Reply:
Here，Before didn't we show warning message back to user or we directly use Maxvalue instead?

Reply:
No warning here, we directly set to max.